sync_labels.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python3
+import sys
+import os
+import argparse
+from github import Github, GithubException
+
+"""
+Sync Github Labels
+
+Synchronize labels idempotently between two Github repositories.
+"""
+
+
+def sync_labels(src_repo, dest_repo, dry_run, delete):
+    """
+    Main subroutine: sync labels between source/destination Github repos
+
+    :param src_repo: full repo name for source of labels
+    :param dest_repo: full repo name for destination of label actions
+    :param dry_run: do a dry run, do not perform the actions
+    :param delete: delete labels that are in destination but not in source
+    """
+
+    # Get PyGithub objects representing repos
+    g = Github(get_access_token())
+    try:
+        s = g.get_repo(src_repo)
+    except GithubException:
+        raise Exception(f"Source repo is not a repo: {src_repo}")
```

Comment:
I think it would probably be more helpful to not catch the GithubException. This way the user gets more information about why the operation failed. If you want to add more information, logging before re-raising could be helpful.

Reply:
Addressed in 08ff939